packages/clay-css/src/scss/components/_empty-state.scss

```diff
@@ -63,3 +63,132 @@
 		@include clay-css(map-get($c-empty-state-sm, c-empty-state-footer));
 	}
 }
+
+// Empty State Animation
+.c-empty-state-animation {
+	.empty-state-animation {
+		@media (prefers-reduced-motion: no-preference) {
+			.ship {
+				transform-origin: bottom;
+				animation: 6s ease-in-out infinite ship;
+			}
+			.wave {
+				transform-origin: center;
+				transform-box: content-box;
+				animation: 3s ease-out infinite wave;
+			}
+			@keyframes ship {
+				50% {
+					transform: translate(0, 25px);
+				}
+				100% {
+					transform: translate(0, 0);
+				}
+			}
+			@keyframes wave {
+				80% {
+					transform: scale(5);
+					opacity: 0;
+				}
+				81% {
+					transform: scale(1);
+					opacity: 0;
+				}
+			}
+		}
+		.c-prefers-reduced-motion .ship,
+		.c-prefers-reduced-motion .wave {
```

Comment:
This selector should be outside of `.empty-state-animation`. It should look like:
```
.c-empty-state-animation {
  &.c-prefers-reduced-motion,
  .c-prefers-reduced-motion {
    .ship,
    .wave {}
  }
}
```

Reply:
Hey @pat270, thanks for pointing that out! 
Indeed the reduced motion was not working on the SVGs and after this change, it is still not working (I can still see the animation).

Should we make it more specific? Ex.:
```css
.c-empty-state-animation {
  &.c-prefers-reduced-motion,
  .c-prefers-reduced-motion .empty-state-svg {
    .ship,
    .wave {}
  }
}
```

Reply:
Tested my suggestion and didn't work either...

Reply:
```
	&.c-prefers-reduced-motion,
	.c-prefers-reduced-motion {
		.empty-state-animation {
			.ship,
			.wave {
				animation: none;
			}
		}
	}
```
is working for me. You can put it at the end of the Sass statement too.